conf/app.routes

```diff
@@ -29,3 +29,11 @@ POST        /new-website-address               controllers.CaptureWebsiteControl
 # confirm website
 GET         /confirm-new-website-address       controllers.ConfirmWebsiteController.show
 POST        /confirm-new-website-address       controllers.ConfirmWebsiteController.updateWebsite
+
+GET         /confirm-website                   controllers.ConfirmWebsiteController.show
```

Comment:
These two routes already exist above this. Not sure why they have been re-added here?